azerothcore/azerothcore-wotlk
# fix(DB/SAI): Quest "Jack Likes His Drink"

Repair gossip menus for "Jack Likes His Drink" quest


##### CHANGES PROPOSED:

-  Update smart_scripts table for Olga and Jack to reflect proper gossip menus
-  Update character_template for Jack to point to correct gossip menu


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1854


##### TESTS PERFORMED:


Applied changes to database then ran through entire quest. Debt item was properly acquired and quest was able to be completed successfully.

##### HOW TO TEST THE CHANGES:


Go to pirate encampment in Howling Fjord and acquire quest "Jack Likes His Drink". Talk to Olga and give her a gold piece to deliver a spiked drink to Jack. Jack will go down and you should be able to click on him to collect the quest item needed. Turn in.

##### KNOWN ISSUES AND TODO LIST:



##### Target branch(es):

Master
